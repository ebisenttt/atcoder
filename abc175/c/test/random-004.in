-976168329686451 637413237735570 882548961468927
