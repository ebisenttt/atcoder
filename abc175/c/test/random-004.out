788929593251403
